staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -228,7 +228,7 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	case len(subresource) == 0:
 		handler = r.serveResource(w, req, requestInfo, crdInfo, terminating, supportedTypes)
 	default:
-		http.Error(w, "the server could not find the requested resource", http.StatusNotFound)
+		http.Error(w, fmt.Sprintf("the server could not find the requested resource %s", crdName), http.StatusNotFound)
```

Comment:
it has to include the subresource, doesn't it?

Reply:
> it has to include the subresource, doesn't it?

we also need to include the resource. we have the resource and subresource extracted from the request.

we could do something like this:
```
	msg := fmt.Sprintf("the server could not find the requested resource: %s/%s", resource, subresource)
	if len(subresource) == 0 {
		msg = fmt.Sprintf("the server could not find the requested resource: %s", resource)
	}
```

Wdyt?